### DISCUSSION TITLE
New base model tags added to CivitAI

### DISCUSSION THREAD

**Discussion by ManOrMonster** [BlafKing/sd-civitai-browser-plus#209] [Ideas]
CivitAI has added "Stable Cascade" as a new base model.

**Comment by ManOrMonster** [comment]
Also, "SDXL Lightning" and "Pony"

<img width="294" alt="image" src="https://github.com/BlafKing/sd-civitai-browser-plus/assets/3925151/09f78044-e621-4139-986d-747894af871a">

**Comment by BlafKing** [comment]
I've now included these in the latest version!

Thanks for letting me know :)
